launch/zed_main.launch

```diff
@@ -45,4 +50,34 @@
         <arg name="aruco_size"  value="$(arg aruco_size)"/>
     </include>
 
+    <node name="classification_splitter" pkg="infra" type="split_measurement.py" output="screen" ></node>
```

Comment:
Go through the scripts for these nodes, they may contain something we want to keep. Add to this PR if something worth to keep